tools/precompute/mc_meta/LookUpTable.h

```diff
@@ -2319,4 +2319,2327 @@ static const char casesClassic[256][16] = {
 
 
 
+#endif // _LOOKUPTABLE_H_
+=======
+/**
+ * @file    LookUpTable.h
+ * @author  Thomas Lewiner <thomas.lewiner@polytechnique.org>
+ * @author  Math Dept, PUC-Rio
+ * @version 0.2
+ * @date    12/08/2002
+ *
+ * @brief   LookUpTable for the MarchingCubes 33 Algorithm
```

Comment:
I'm probably missing something, but right now it looks like this addition just duplicates the content of the file that's already there? The line length of the file is almost doubled exactly and I can't find a difference between the addition and what's already there...

Reply:
Yep, I've deleted the duplicated second half and checked that it's now identical with the same file in the main branch of scikit-image.

Reply:
No worries. Merge conflicts can get a bit messy. :sweat_smile: